aci/resource_aci_bgppeerp.go

```diff
@@ -623,7 +621,13 @@ func resourceAciBgpPeerConnectivityProfileUpdate(ctx context.Context, d *schema.
 	}
 
 	PeerConnectivityProfileDn := bgpPeerP.DistinguishedName
-	if _, ok := d.GetOk("local_asn"); ok {
+	if d.HasChange("local_asn") || d.HasChange("local_asn_propagate") {
+
+		err := aciClient.DeleteByDn(dn+"/localasn", "bgpLocalAsnP")
```

Comment:
should we point to models.BgplocalasnpClassName in bgp_local_asn_p.go of aci client?